### PR TITLE
Add Geodata for Rottnest Island (Australia)

### PR DESCRIPTION
**Pull request by ajoga**
This enable that island to have its own data collected and published online. The PR for the parser is dependant on this PR ([see here](https://github.com/tmrowco/electricitymap-contrib/pull/2867#issuecomment-771591341)). Both of these PR will solve #2534 .

I took inspiration from #2533 

To core reviewer, please see my inline comment about one change I do not fully understand that I had to make.

### CODE REVIEW

File: web/generate-geometries.js [hunk 1]
```diff
@@ -947,6 +949,6 @@ topoMoreDetails = topojson.simplify(topoMoreDetails, 0.001);
 // Merge topoMoreDetails into topo
 mergeTopoJsonSingleZone(topo, topoMoreDetails);
 
-topo = topojson.filter(topo, topojson.filterWeight(topo, 0.009));
+//topo = topojson.filter(topo, topojson.filterWeight(topo, 0.009));
```
Comment (ajoga) [inline web/generate-geometries.js:952]: The island is very small (19km2).

With the line commented, I get the island to render in the browser, with `"AUS-WA-RI":{"type":"MultiPolygon","arcs":[[[9507]]]}` in `/web/src/world.json` (size=641K)

With the line uncommented, the island does not appear in the browser and gets referred to as `"AUS-WA-RI":{"type":null}` in `/web/src/world.json` (size=547K)

I guess this code serves some purpose to reduce the size of the map to be made available to the browser, but I don't know what to do with it so this island can be rendered. You'll note that `moreDetails: true` doesn't help.

**Comment by madsnedergaard** [comment]
Hey @ajoga, thanks for the work here!

You're right that it's there to reduce the file-size, but it should only apply to zones that don't have the `moreDetails` flag :)

I have a fix for this in #2987, and using that the geometries for Rottnest Island are generated:

Screenshot 2021-02-09 at 16 49 33

It's not the most pretty shape, but to fair it is a really small island! 😄 

Once the other PR is un-merged, you should be able to revert the uncommenting in `generate-geometries` and we can get this merged :)

File: web/generate-geometries.js (hunk 1 above)
Comment (ajoga) [reply web/generate-geometries.js]: addressed by @madsnedergaard in #2987 :)

**Review by madsnedergaard — APPROVED** [review]
Awesome! 💪

**Pull request merged**